README.md

```diff
@@ -18,210 +18,181 @@ Darkflame Universe is licensed under AGPLv3, please read [LICENSE](LICENSE). Som
 
-```bash
-# Create the build directory, preserving it if it already exists
-mkdir -p build
-cd build
+For Ubuntu, you would run the following commands. On other systems, the package install command will differ.
 
-# Run CMake to generate make files
-cmake ..
+```bash
+sudo apt update && sudo apt upgrade
 
-# To build utilizing multiple cores, append `-j` and the amount of cores to utilize, for example `cmake --build . --config Release -j8'
-cmake --build . --config Release
+# Install packages
+sudo apt install build-essential gcc zlib1g-dev libssl-dev openssl mariadb-server cmake
 ```
 
-### MacOS builds
-Ensure `cmake`, `zlib` and `open ssl` are installed as well as a compiler (e.g `clang` or `gcc`).
-
-In the repository root folder run the following. Ensure -DOPENSSL_ROOT_DIR=/path/to/openssl points to your openssl install location
+#### Required CMake version
+This project uses <font size="4">**CMake version 3.18**</font> or higher and as such you will need to ensure you have this version installed.
+You can check your CMake version by using the following command in a terminal.
 ```bash
-# Create the build directory, preserving it if it already exists
-mkdir -p build
-cd build
-
-# Run CMake to generate build files
-cmake .. -DOPENSSL_ROOT_DIR=/path/to/openssl
-
-# Get cmake to build the project. If make files are being used then using make and appending `-j` and the amount of cores to utilize may be preferable, for example `make -j8`
-cmake --build . --config Release
+cmake --version
 ```
 
-### Windows builds (native)
-Ensure that you have either the [MSVC](https://visualstudio.microsoft.com/vs/) or the [Clang](https://github.com/llvm/llvm-project/releases/) (recommended) compiler installed. You will also need to install [CMake](https://cmake.org/download/). Currently on native Windows the server will only work in Release mode.
+If you are going to be using an ubuntu environment to run the server, you may need to get a more recent version of `cmake` than the packages available may provide.
 
-#### Build the repository
-```batch
-:: Create the build directory
-mkdir build
-cd build
+The general approach to do so would be to obtain a copy of the signing key and then add the CMake repository to your apt.
+You can do so with the following commands.
 
-:: Run CMake to generate make files
-cmake ..
+[Source of the below commands](https://askubuntu.com/questions/355565/how-do-i-install-the-latest-version-of-cmake-from-the-command-line)
 
-:: Run CMake with build flag to build
-cmake --build . --config Release
-```
-#### Windows for ARM has not been tested but should build by doing the following
-```batch
-:: Create the build directory
-mkdir build
-cd build
+```bash
+# Remove the old version of CMake
+sudo apt purge --auto-remove cmake
 
-:: Run CMake to generate make files
-cmake .. -DMARIADB_BUILD_SOURCE=ON
+# Prepare for installation
+sudo apt update && sudo apt install -y software-properties-common lsb-release && sudo apt clean all
```

Comment:
That's a typical stackoverflow thing. Ask people to install two new packages, just so the next command to run is slightly more generic.

IMO it would be better to just tell people to add a file to /etc/apt/sources.list.d and find out the ubuntu version name on their own (or tell them it's `jammy` now. iirc)

Reply:
Will make an issue for this so we can get rid of this eventually.